src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Stop: require('./circuit/hop'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
+}
```

Comment:
Timing wise it's a bit unfortunate -- we've been discussing the circuit relay explicitly as a transport for months, and you've given your +1 multiple times. What has changed?